standalone-metastore/pom.xml

```diff
@@ -113,7 +113,7 @@
     <jetty.version>9.4.45.v20220203</jetty.version>
     <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
     <!-- If upgrading, upgrade atlas as well in ql/pom.xml, which brings in some springframework dependencies transitively -->
```

Comment:
The new atlas will be out soon
https://lists.apache.org/thread/ro0fhrqknzzm6tvxps3d3pclb86lfphf